src/DataType.luna

```diff
@@ -14,5 +14,6 @@ class DataType:
 
     def name: self.ptr.name
     def toText: self.ptr.toText
+    def shortRep: "DataType"
```

Comment:
Is there no more information that can be provided here?

Reply:
Consider using `toText` to obtain Arrow-formated string with the complete type info — not sure however if it is pretty enough (and consistent in naming).
Otherwise API is not complete enough — `name` can provide the name for this type but not its type arguments (so you'll get `vector` when you have `vector int` etc.)